lib/fastlane/plugin/test_center/helper/test_collector.rb

```diff
@@ -25,6 +25,8 @@ def initialize(options)
         if @batch_count == 1 && options[:parallel_testrun_count] > 1
           @batch_count = options[:parallel_testrun_count]
         end
+
+        @test_prefix = options[:test_prefix] || "test"
```

Comment:
If you put in the validation in the ConfigItem in `multi_scan`, do you need this? I don't mind it: I am asking you and the Universe (my subconscious lives there).

Reply:
You're totally right - this is redundant and can be removed.